tests/VolleyManagement.UnitTests/Services/GameReportService/Comparers/PivotStandingsComparer.cs

```diff
@@ -6,6 +6,26 @@
 
     internal class PivotStandingsComparer : IComparer<PivotStandingsDto>
     {
+        private IComparer<TeamStandingsDto> teamsComparer;
+        private bool gameResultComparer = false;
```

Comment:
private variaables should start with underscore and bool variables should start with "is/are/does/should" to indicate condition